pytorch-notebook/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) Jupyter Development Team.
+# Distributed under the terms of the Modified BSD License.
+ARG OWNER=jupyter
+ARG BASE_CONTAINER=$OWNER/scipy-notebook
+FROM $BASE_CONTAINER
+
+LABEL maintainer="Jupyter Project <jupyter@googlegroups.com>"
+
+# Fix: https://github.com/hadolint/hadolint/wiki/DL4006
+# Fix: https://github.com/koalaman/shellcheck/wiki/SC3014
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+
+# Install PyTorch with pip
+# hadolint ignore=DL3013
+RUN pip install --no-cache-dir --index-url 'https://download.pytorch.org/whl/cpu' \
```

Comment:
Do we really need to add index-url here?

Reply:
Yes, leaving it out will install cuda-dependencies.

The index-url is supplied by the official installation instructions on https://pytorch.org/get-started/locally/:

![image](https://github.com/jupyter/docker-stacks/assets/1088931/605e2cf2-ff45-4069-82d2-51aece01987c)

Reply:
Great, thanks for the answer.